plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinitionWidget.tsx

```diff
@@ -134,12 +133,46 @@ type Props = {
   definition: string;
 };
 
+const fetchResolver = {
+  order: 199,
+  canReadValue: false,
+  canRead(file: any) {
+    if ((this.canReadValue as any) instanceof Boolean) {
+      return this.canReadValue;
+    }
+    return file.url.match(new RegExp((this.canReadValue as unknown) as string));
+  },
+  fetchOptions: {},
+
+  async read(file: any) {
+    // @ts-ignore
```

Comment:
Avoid using @ts-ignore if possible, it's better to cast to `any`. What is it that is being ignored here?